strongbox-web-core/src/main/java/org/carlspring/strongbox/config/SecurityConfig.java

```diff
@@ -74,24 +74,24 @@ public void init(WebSecurity web)
     protected void configure(HttpSecurity http)
             throws Exception
     {
-        http.sessionManagement()
-            .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
-            .and()
+        http
+            .addFilterBefore(strongboxAuthenticationFilter(),
```

Comment:
Could you merge this line with the one above and re-align all the ones below? Thanks!